diff --git "a/C:\\Users\\sirok\\MoCKA\\inventory\\phase6_enforcer_20260221_053219\\target.before" "b/C:\\Users\\sirok\\MoCKA\\inventory\\phase6_testfiles\\dummy_target.txt"
index 925d96c..0519b0e 100644
--- "a/C:\\Users\\sirok\\MoCKA\\inventory\\phase6_enforcer_20260221_053219\\target.before"
+++ "b/C:\\Users\\sirok\\MoCKA\\inventory\\phase6_testfiles\\dummy_target.txt"
@@ -1 +1,2 @@
 ・ｿalpha
+beta
